GameServerLib/Scripting/CSharp/CSharpScriptEngine.cs

```diff
@@ -173,6 +173,7 @@ public T CreateObject<T>(string scriptNamespace, string scriptClass)
                 return default(T);
             }
 
+            scriptNamespace = scriptNamespace.Replace(" ", "_");
```

Comment:
The scripts (in namespaces that contain spaces) didn't work before this change?

Reply:
They worked, however this change makes it so the name of the spell in content is properly assigned to the script.